index.js

```diff
@@ -0,0 +1,156 @@
+        now = true;
+    }
+
+    if (normalizedMessage.indexOf('weather tomorrow') !== -1) {
+        tomorrow = true;
+    }
+
+    if (!now && !tomorrow) {
+        if (rtmConnectionOpened) {
+            if (normalizedMessage.indexOf('help') !== -1) {
+                rtm.sendMessage(`<@${message.user}>, try @surry-interview 'weather now' or 'weather tomorrow'`, botChannel);
+            } else {
+                rtm.sendMessage(`<@${message.user}>, I did not understand that command, try @surry-interview 'weather now' or 'weather tomorrow'`, botChannel);
+                console.log('Ignoring unknown command: ' + normalizedMessage.substring(0, 20) + '...');
+            }
+        }
+        return;
+    }
+
+    if (now) {
+        util.getCurrentWeatherReport().then((json) => {
+            if (rtmConnectionOpened) {
+                if (json.currently) {
+                    let message = util.generateCurrentSummary(json.currently);
+                    rtm.sendMessage(message, botChannel);
+
+                    const now = new Date().getTime();
+                    // This url can be used to grab a frame from a DDOT (DC) traffic cam!
+                    // This can be used to give a rough visual of the weather. The camera is
+                    // at the intersection of 14th St and Independence St NW. Looks like the current
+                    // time in milliseconds since 1970 needs to be passed in to get the latest images.
+                    // The RTM client doesn't allow sending attachments (see https://api.slack.com/rtm),
+                    // but the Slack URL unfurling functionality automatically includes the traffic cam frame!
+                    // See http://app.ddot.dc.gov/ for more info.
+                    const trafficCamUrl = `http://ie.trafficland.com/v1.0/200146/full?system=ddot&pubtoken=b4c0b819f66741b99fdb17c963f47e3bafd09b494019ee9624f1d38641ebc6bf&refreshRate=2000&t=${now}`
+
+                    // the RTM client unfortunately doesn't support formatting of URLs with <URL|human-friendly-string>
+                    rtm.sendMessage(`Here's a snapshot of the current weather at 14th St and Independence St in DC: ${trafficCamUrl}`, botChannel);
+                }
+            }
+        }).catch((error) => {
+            console.error(error);
+        });
+    }
+
+    if (tomorrow) {
+        util.getTomorrowsWeatherReport().then((json) => {
+            if (rtmConnectionOpened) {
+                if (json.daily.data.length >= 1) {
+                    let message = util.generateTomorrowsSummary(json.daily.data[0]);
+                    rtm.sendMessage(message, botChannel);
+                } else {
+                    rtm.sendMessage('There is no forecast for tomorrow! The end is nigh.', botChannel);
+                }
+            }
+        }).catch((error) => {
+            console.log(error);
+        });
+    }
+});
+
+let dailyAnnounceJob;
+let currentWeatherIcon;
```

Comment:
Yeah, I'd need to start storing the previous day's weather for multiple locations and/or issue a request for yesterday's weather using Darksky's Time Machine request so that I could notify the channel of a change in weather for any location.

I'd also need to update the message parsing code to recognize multiple locations by zip code/city, and then map those locations to a latitude/longitude tuple for Darksky's API.